common/src/main/java/com/wynntils/models/lootruns/LootrunRenderer.java

```diff
@@ -205,4 +282,75 @@ private static void renderPoint(VertexConsumer consumer, Matrix4f lastMatrix, Co
                 .normal(0, 0, 1)
                 .endVertex();
     }
+
+    private static void renderTexturedQueuedPoints(
+            List<Pair<ColoredPoint, ColoredPoint>> pointPairList, PoseStack poseStack, VertexConsumer vertexConsumer) {
+        for (Pair<ColoredPoint, ColoredPoint> pointPair : pointPairList) {
+            renderTexturedPoint(pointPair.a(), pointPair.b(), poseStack, vertexConsumer);
+        }
+    }
+
+    private static void renderTexturedPoint(
+            Pair<ColoredPoint, ColoredPoint> pointPair, PoseStack poseStack, VertexConsumer vertexConsumer) {
+        renderTexturedPoint(pointPair.a(), pointPair.b(), poseStack, vertexConsumer);
+    }
+
+    private static void renderTexturedPoint(
+            ColoredPoint start, ColoredPoint end, PoseStack poseStack, VertexConsumer vertexConsumer) {
+        Vec3 camPos = McUtils.mc().gameRenderer.getMainCamera().getPosition();
+        Vec3 startVec = start.vec3();
+        Vec3 endVec = end.vec3();
+        int color = start.color();
+
+        // vertex position delta to starting point
+        Vector3f pos1 = new Vector3f(-0.5f, 0.24f, -0.5f);
+        Vector3f pos2 = new Vector3f(0.5f, 0.24f, -0.5f);
+        Vector3f pos3 = new Vector3f(0.5f, 0.24f, 0.5f);
+        Vector3f pos4 = new Vector3f(-0.5f, 0.24f, 0.5f);
+
+        Vec3 direction =
+                new Vec3(endVec.x, endVec.y, endVec.z).subtract(startVec).normalize();
+
+        // rotation angle to point surface normal to end position
+        // 1.5707963267948966 is the result of Math.toRadians(90f), so the arrow point to the end position instead
+        float xAngle = (float) ((float) Math.acos(direction.y / direction.length()) - 1.5707963267948966);
```

Comment:
You can use Math.PI / 2 and it will look nicer :) (and the compiler will optimize for it)